server/parsec/components/postgresql/__init__.py

```diff
@@ -1,6 +1,11 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 from __future__ import annotations
 
+import asyncpg
+
+type PgConn = asyncpg.Connection[asyncpg.Record]
+type PgPool = asyncpg.Pool[asyncpg.Record]
```

Comment:
Same here. 

I would call them `Connection` and `Pool` but I guess it's a matter of taste?

Reply:
done